Update from staged recipes

leofang opened the pull request:
Follow-up of conda-forge/staged-recipes#17550.

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

leofang committed:
check in latest change

MNT: Re-rendered with conda-build 3.21.6, conda-smithy 3.16.2, and conda-forge-pinning 2022.01.28.10.42.27

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

leofang committed:
add ppc64le support

MNT: Re-rendered with conda-build 3.21.6, conda-smithy 3.16.2, and conda-forge-pinning 2022.01.28.10.42.27

leofang commented:
Blocked by https://github.com/conda-forge/cuquantum-feedstock/pull/5.

leofang committed:
disable ppc64le builds for now

MNT: Re-rendered with conda-build 3.21.6, conda-smithy 3.16.2, and conda-forge-pinning 2022.01.28.19.47.10

leofang committed:
fix
